src/Subtitles/Rasterizer.cpp

```diff
@@ -27,6 +27,34 @@
 #include "Rasterizer.h"
 #include "SeparableFilter.h"
 #include "../SubPic/ISubPic.h"
+#include <ft2build.h>
+#include <freetype/ftoutln.h>
+#include <freetype/internal/ftobjs.h>
+#include FT_FREETYPE_H
+#include FT_SYNTHESIS_H
+
+#define DEBUG_PERFORMANCE 0
+#if DEBUG_PERFORMANCE
+#include <sstream>
+#include <chrono>
+#define BEGIN_PERF_TIMER(a) \
+    std::chrono::steady_clock::time_point begin##a = std::chrono::steady_clock::now();
+
+#define END_PERF_TIMER(a, ref1, ref2) \
+    std::chrono::steady_clock::time_point end##a = std::chrono::steady_clock::now(); \
+    { \
+        std::ostringstream ss; \
+        ss << "\n"; \
+        ss << std::chrono::time_point_cast<std::chrono::microseconds>(end##a).time_since_epoch().count(); \
+        ss << ": "##ref1 << "(" << CW2A(ref2) << ") = "; \
+        ss << std::chrono::duration_cast<std::chrono::microseconds>(end##a - begin##a).count() << "[�s]" << std::endl; \

```

Comment:
github shows unsupported unicode char in this line?

Reply:
Must be the micro.

Reply:
Weird thing is that it is displayed correctly in my quote. Probably just a github glitch.